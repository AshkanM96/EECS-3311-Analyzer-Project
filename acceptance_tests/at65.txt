conjoin
conjoin
boolean_constant(true)
conjoin
boolean_constant(true)
boolean_constant(false)
conjoin
boolean_constant(true)
boolean_constant(true)
type_check
evaluate
