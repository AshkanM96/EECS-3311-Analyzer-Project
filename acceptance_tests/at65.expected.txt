  Expression currently specified: ?
  Report: Expression is initialized.
->conjoin
  Expression currently specified: (? && nil)
  Report: OK.
->conjoin
  Expression currently specified: ((? && nil) && nil)
  Report: OK.
->boolean_constant(True)
  Expression currently specified: ((True && ?) && nil)
  Report: OK.
->conjoin
  Expression currently specified: ((True && (? && nil)) && nil)
  Report: OK.
->boolean_constant(True)
  Expression currently specified: ((True && (True && ?)) && nil)
  Report: OK.
->boolean_constant(False)
  Expression currently specified: ((True && (True && False)) && ?)
  Report: OK.
->conjoin
  Expression currently specified: ((True && (True && False)) && (? && nil))
  Report: OK.
->boolean_constant(True)
  Expression currently specified: ((True && (True && False)) && (True && ?))
  Report: OK.
->boolean_constant(True)
  Expression currently specified: ((True && (True && False)) && (True && True))
  Report: OK.
->type_check
  Expression currently specified: ((True && (True && False)) && (True && True))
  Report: ((True && (True && False)) && (True && True)) is type-correct.
->evaluate
  Expression currently specified: ((True && (True && False)) && (True && True))
  Report: False
